src/scvi/dataloaders/_data_splitting.py

```diff
@@ -34,27 +38,77 @@ def validate_data_split(
         Size of train set. Need to be: 0 < train_size <= 1.
     validation_size
         Size of validation set. Need to be 0 <= validation_size < 1
+    use_external_indexing
+        Whether to use external supported indexing. This bypass any other flag or input parameter
+        that was before
+    external_indexing
+        A list of np.arrays that is always in the order of [[train_idx],[valid_idx],[test_idx]].
```

Comment:
```suggestion
        A list of data split indexes in the order of training, validation, and test sets.
```
nit: Since our docs will render type annotations directly, we can remove type references in argument docstrings for less redundancy.

Reply:
We should also mention that validation and test are not required

Reply:
gotcha. done.